### DISCUSSION TITLE
Pedido de reembolso

### DISCUSSION THREAD

**Discussion by biabarsande** [aclap-dev/video-downloadhelper#1580] [License & Purchase requests]
### What's your issue?

I purchased it and found it difficult to use, so I want to cancel the purchase.

### The 10 first digits (only 10) of your license number or email address

a5d810d39b

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by jcfrog** [comment]
Hi,
You have been refunded. Should be available in a few business days.
